modules/apps/ccv/types/expected_keepers.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+// StakingKeeper defines the contract expected by parent-chain ccv module.
+// StakingKeeper is responsible for keeping track of latest validator set of all baby chains
+type StakingKeeper interface {
```

Comment:
These are the responsibilities of the staking module. This is the only interface CCV will expect. Staking module may make the logic as complex or simple as they like. This will allow upgrades in cross-chain delegation functionality to occur with staking module upgrades without affecting CCV's IBC module